frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -68,6 +91,26 @@ async function deleteUserRoleAssignment(
         );
     }
 }
+
+const deleteAppAdmin = async ( admin: FamAppAdminGet) => {
```

Comment:
you might want to try format the file again, just left some small pieces like the space here

Reply:
that one is on me, I will  fix it